tests/idempotency/IdempotencyCheck.scala

```diff
@@ -6,7 +6,12 @@ import java.util.stream.Stream as JStream
 import scala.collection.JavaConverters.*
 
 object IdempotencyCheck {
-  val blacklisted = Set(
+  val flakytests =
+    if scala.util.Properties.isWin
+    then Set(s"pos{JFile.separator}i6507b")
```

Comment:
```suggestion
    then Set(s"pos${JFile.separator}i6507b")
```

The blacklist bellow seems to have the same issue. Maybe we should handle these paths as `JPath` to avoid the `\` vs `/` complications.

Reply:
Actually, we can try to remove the blacklist below, as they are not effective.

Reply:
I meant not to use the `toString` in the first place

Reply:
This can be improve in a subsequent PR